File: TugasPraktikumModul2.java
//Membuat Program Privat Online

package com.tutorial;
import java.util.Scanner;

public class TugasModul2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Selamat datang di website kami. Silakan masukkan nama Anda");
        System.out.print("Nama : ");
        String nama = input.nextLine();
        System.out.println("Hallo " + nama);

        int jenjang;
        System.out.println("1. SD");
        System.out.println("2. SMP");
        System.out.println("3. SMA");
        System.out.print("Silakan pilih jenjang Anda dengan memilih nomor 1-3 : ");
        jenjang = input.nextInt();

        if(jenjang == 1){
            System.out.println("1. Matematika");
            System.out.println("2. IPA");
            System.out.println("3. Bahasa Inggris");
            System.out.print("Silakan pilih pelajaran dengan memilih nomor 1-3 : ");
            int pelajaran = input.nextInt();
            switch(pelajaran){
                case 1 :
                    System.out.println("Kami akan mencarikan guru privat Matematika SD untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                case 2 :
                    System.out.println("Kami akan mencarikan guru privat IPA SD untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                case 3 :
                    System.out.println("Kami akan mencarikan guru privat Bahasa Inggris SD untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                default :
                    System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
            }
        }else if(jenjang == 2){
            System.out.println("1. Matematika");
            System.out.println("2. IPA");
            System.out.println("3. Bahasa Inggris");
            System.out.print("Silakan pilih pelajaran dengan memilih nomor 1-3 : ");
            int pelajaran = input.nextInt();
            switch(pelajaran){
                case 1 :
                    System.out.println("Kami akan mencarikan guru privat Matematika SMP untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                case 2 :
                    System.out.println("Kami akan mencarikan guru privat IPA SMP untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                case 3 :
                    System.out.println("Kami akan mencarikan guru privat Bahasa Inggris SMP untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                    break;
                default :
                    System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
            }
        }else if(jenjang == 3){
            System.out.println("1. IPA");
            System.out.println("2. IPS");
            System.out.print("Silakan pilih jurusan dengan memilih nomor 1-2 : ");
            int jurusan = input.nextInt();
            if(jurusan == 1){
                System.out.println("1. Matematika");
                System.out.println("2. Fisika");
                System.out.println("3. Biologi");
                System.out.print("Silakan pilih pelajaran dengan memilih nomor 1-3 : ");
                int pelajaran = input.nextInt();
                switch(pelajaran){
                    case 1 :
                        System.out.println("Kami akan mencarikan guru privat Matematika untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    case 2 :
                        System.out.println("Kami akan mencarikan guru privat Fisika untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    case 3 :
                        System.out.println("Kami akan mencarikan guru privat Biologi untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    default :
                        System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
                }
            }else if (jurusan == 2){
                System.out.println("1. Ekonomi");
                System.out.println("2. Geografi");
                System.out.println("3. Sosiologi");
                System.out.print("Silakan pilih pelajaran dengan memilih nomor 1-3 : ");
                int pelajaran = input.nextInt();
                switch(pelajaran){
                    case 1 :
                        System.out.println("Kami akan mencarikan guru privat Ekonomi untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    case 2 :
                        System.out.println("Kami akan mencarikan guru privat Geografi untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    case 3 :
                        System.out.println("Kami akan mencarikan guru privat Sosiologi untuk Anda. Pemberitahuan lebih lanjut akan dikirim lewat email Anda");
                        break;
                    default :
                        System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
                }
            }else{
                System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
            }
        }else{
            System.out.println("Kami tidak mengenal angka yang Anda masukkan. Silakan masukkan kembali");
        }
    }
}

File: TugasPraktikumModul2.cpp
//Membuat program daftar menu pada restoran
#include <iostream>
using namespace std;

int main() {

	cout << "Selamat Datang di Restoran Kami" << endl;
	int pilih;
	string menu[5] = {
		"1. Nasi Goreng", "2. Mie goreng", "3. Capcai", "4. Ayam goreng", "5. Bebek goreng"
	};
	cout << "Menu : " << endl;
	cout << menu[0] << endl;
	cout << menu[1] << endl;
	cout << menu[2] << endl;
	cout << menu[3] << endl;
	cout << menu[4] << endl;
	cout << "Silakan pilih menu di atas dengan menekan 1-5 : ";
	cin >> pilih;

	if (pilih == 1) {
		int jumlah, total;
		cout << "Masukkan banyaknya makanan yang akan dipesan : ";
		cin >> jumlah;
		cout << endl;
		total = jumlah * 15000;
		if (jumlah > 10) {
			cout << "Tidak dapat memesan makanan lebih dari 10 buah" << endl;
		}
		else {
			cout << "STRUK PEMBAYARAN" << endl;
			cout << "Makanan yang dipesan : Nasi goreng" << endl;
			cout << "Jumlah pesanan : " << jumlah << endl;
			cout << "Total harga : Rp " << total << endl;
		}
	}
	else if(pilih == 2){
		int jumlah, total;
		cout << "Masukkan banyaknya makanan yang akan dipesan : ";
		cin >> jumlah;
		cout << endl;
		total = jumlah * 14000;
		if (jumlah > 10) {
			cout << "Tidak dapat memesan makanan lebih dari 10 buah" << endl;
		}
		else {
			cout << "STRUK PEMBAYARAN" << endl;
			cout << "Makanan yang dipesan : Mie goreng" << endl;
			cout << "Jumlah pesanan : " << jumlah << endl;
			cout << "Total harga : Rp " << total << endl;
		}
	}
	else if (pilih == 3) {
		int jumlah, total;
		cout << "Masukkan banyaknya makanan yang akan dipesan : ";
		cin >> jumlah;
		cout << endl;
		total = jumlah * 16000;
		if (jumlah > 10) {
			cout << "Tidak dapat memesan makanan lebih dari 10 buah" << endl;
		}
		else {
			cout << "STRUK PEMBAYARAN" << endl;
			cout << "Makanan yang dipesan : Capcai" << endl;
			cout << "Jumlah pesanan : " << jumlah << endl;
			cout << "Total harga : Rp " << total << endl;
		}
	}
	else if (pilih == 4) {
		int jumlah, total;
		cout << "Masukkan banyaknya makanan yang akan dipesan : ";
		cin >> jumlah;
		cout << endl;
		total = jumlah * 18000;
		if (jumlah > 10) {
			cout << "Tidak dapat memesan makanan lebih dari 10 buah" << endl;
		}
		else {
			cout << "STRUK PEMBAYARAN" << endl;
			cout << "Makanan yang dipesan : Ayam goreng" << endl;
			cout << "Jumlah pesanan : " << jumlah << endl;
			cout << "Total harga : Rp " << total << endl;
		}
	}
	else if (pilih == 5) {
		int jumlah, total;
		cout << "Masukkan banyaknya makanan yang akan dipesan : ";
		cin >> jumlah;
		cout << endl;
		total = jumlah * 22000;
		if (jumlah > 10) {
			cout << "Tidak dapat memesan makanan lebih dari 10 buah" << endl;
		}
		else {
			cout << "STRUK PEMBAYARAN" << endl;
			cout << "Makanan yang dipesan : Bebek goreng" << endl;
			cout << "Jumlah pesanan : " << jumlah << endl;
			cout << "Total harga : Rp " << total << endl;
		}
	}
	else {
		cout << "Maaf, kami tidak mengenal angka yang Anda masukkan. Silakan coba kembali" << endl;
	}
	return 0;
}
